rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2464,6 +2464,52 @@ void voteFederationChange_addWrongFederatorPublicKey_returnsGenericResponseCode(
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorMultiKey_returnsSuccessfulResponseCode() {
+
+            // Arrange
+
+            BtcECKey expectedBtcECKey = new BtcECKey();
+            ECKey expectedRskKey = new ECKey();
+            ECKey expectedMstKey = new ECKey();
```

Comment:
Use deterministic values please

Reply:
And also meaningful names, those are the keys of a new federation member. Calling them "expected keys" doesn't give much info

Reply:
Updated.